File: Course_01/Java Language/Estrutura repetitiva/divisao.java
package Repetitivo;

import java.util.Locale;
import java.util.Scanner;

public class divisao {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n, i;
		double num, deno, resultado;
		
		System.out.print("Quantos casos vocês vai digitar? ");
		n = sc.nextInt();
		
		for (i = 0; i < n; i++) {
			System.out.print("Entre com o numerador: ");
			num = sc.nextDouble();
			System.out.print("Entre com o denominador: ");
			deno = sc.nextDouble();
			if (deno == 0) {
				System.out.println("DIVISÃO IMPOSSÍVEL");
			}
			else {
				resultado = num / deno;
				System.out.println("DIVISÃO = " + String.format("%.2f", resultado));
			}
		}
		
		sc.close();
	}

}

File: Course_01/Python/EstruturasRepetitivas/divisao.py
n = int(input("Quantos casos você vai digitar? "))

for i in range(0, n):
    numerador = int(input("Entre com o numerador: "))
    denominador = int(input("Entre com o denominador: "))
    if denominador == 0:
        print("DIVISÃO IMPOSSÍVEL")
    else:
        print(f"Divisão = {numerador / denominador}")